### PR TITLE
BW-629: Display Notifications

### PR DESCRIPTION
**Pull request by jitendraashutec**

### CODE REVIEW

**Comment by milansar** [comment]
@anihalaney , PR is ready for your review

File: projects/shared-library/src/lib/shared/mobile/component/action-bar/action-bar.component.ts [hunk 1]
```diff
@@ -29,10 +31,20 @@ export class ActionBarComponent implements OnDestroy {
     constructor(
         private routerExtensions: RouterExtensions,
         public store: Store<CoreState>,
+        public cd: ChangeDetectorRef
     ) {
         this.subscriptions.push(store.select(coreState).pipe(select(s => s.user)).subscribe(user => {
             this.user = user;
         }));
+
```
Comment (anihalaney) [inline projects/shared-library/src/lib/shared/mobile/component/action-bar/action-bar.component.ts]: repeated code?

File: projects/shared-library/src/lib/environments/bitwiser-edu/environment.dev.ts [hunk 2]
```diff
@@ -17,7 +17,7 @@ export const CONFIG: IConfig = {
     messagingSenderId: '704157888306'
   },
   'functionsUrl': `https://bitwiser-edu.firebaseapp.com/${appConstants.API_VERSION}`,
-  'editorUrl': 'https://bitwiser-edu.firebaseapp.com/trivia-editor',
+  'editorUrl': 'https://bitwiser-edu.firebaseapp.com/editor',
```
Comment (anihalaney) [inline projects/shared-library/src/lib/environments/bitwiser-edu/environment.dev.ts]: i feel this path changes with every PR. what's the right path? can we fix it.

Comment (jitendraashutec) [reply projects/shared-library/src/lib/environments/bitwiser-edu/environment.dev.ts]: Yes, we have changed path from trivia-editor to editor

File: projects/shared-library/src/lib/shared/mobile/component/action-bar/action-bar.component.ts (hunk 1 above)
Comment (jitendraashutec) [reply projects/shared-library/src/lib/shared/mobile/component/action-bar/action-bar.component.ts]: I have removed duplicate code.

**Pull request merged**